HHS/Head-Start-TTADP
# [TTAHUB-1754] Training Reports Landing

## Description of change

This change adds the Training Report landing page. We show events under three tab categories Not started, In progress, and Complete.

## How to test

- Review the code.
- View the Training Reports page. 
- Clicking the various tabs should send sync requests for the related data.
- Make sure all acceptance criteria is met from the JIRA.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1754


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
